inc/ix/ethdev.h

```diff
@@ -222,7 +222,7 @@ struct ix_rte_eth_rss_conf {
 #define ETH_RSS_IPV6_UDP    0x0080 /**< IPv6/UDP packet. */
 #define ETH_RSS_IPV6_UDP_EX 0x0100 /**< IPv6/UDP with extension headers. */
 /* Definitions used for redirection table entry size */
-#define ETH_RSS_RETA_NUM_ENTRIES 128
+#define ETH_RSS_RETA_NUM_ENTRIES 512
 #define ETH_RSS_RETA_MAX_QUEUE   16
```

Comment:
Is `ETH_RSS_RETA_MAX_QUEUE` IX-specific or related to the driver? It is used here:
https://github.com/aroulin/ix/blob/i40e-driver-release/dp/core/ethdev.c#L100
 
i40e DPDK driver says `dev_info.max_rx_queues` is 320 which is quite higher than 16.